src/App.jsx

```diff
@@ -1,5 +1,36 @@
 import 'bulma/css/bulma.css';
+import cn from 'classnames';
+
 import './App.scss';
+import { useState } from 'react';
+
+const SORT_ALPHABETICALLY = 'alpabet sort';
+const SORT_BY_LENGTH = 'length sort';
+
+function getChangedGoods(goods, sortType, isReversed) {
```

Comment:
the purpose of the function is to sort, so you can describe it that way
```suggestion
function getSortedGoods(goods, sortType, isReversed) {
```